src/SaveHandler/DbTableGatewayOptions.php

```diff
@@ -9,6 +9,8 @@
 
 /**
  * DbTableGateway Save Handler Options
+ *
+ * @psalm-suppress MissingTemplateParam
```

Comment:
Please remove the inline suppression here and instead add it to the baseline

Reply:
done